queries.js

```diff
@@ -66,6 +76,49 @@ function addProduct(ctx) {
   });
 }
 
+async function getOrdersServedByEmployee(ctx) {
+  const session = driver.session();
+  const { name } = ctx.params;
+  const statement = `MATCH (:Employee {name: '${name}'})-[:ENTERED]->(o:Order) RETURN o`
```

Comment:
Lägg till managerId och startDate och endDate som query parameters så att den bara hämtar från locations som managerId jobbar på, och filtrera så det är "for a specified time period".

MATCH (man: Employer)-[:WORKS_AT]-(:CoffeeShop)<-[:WORKS_AT]-(:Employee {name: '${name}'})-[:ENTERED]->(o:Order) RETURN o